.github/workflows/build-docs.yaml

```
@@ -0,0 +1,17 @@
+name: "Build docs"
+
+on: pull_request
+
+jobs:
+    docs:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: ammaraskar/sphinx-action@master
+              with:
+                  docs-folder: "./"
+                  build-command: "sphinx-build -nW --keep-going -b html -d _build/doctrees . _build/html"
```

would you mind to separate the `-nW` into `-n -W`? i just searched for a `-nW` option for about 10 minutes before i understood that these are two options 🙈 

```suggestion
                  build-command: "sphinx-build -n -W --keep-going -b html -d _build/doctrees . _build/html"
```

---

of course 😄